### DISCUSSION TITLE
request limitiations on https://aip.dfs.de/ ?

### DISCUSSION THREAD

**Discussion by derguenner** [mipastgt/AIPBrowserDE-Public#20] [General]
Wenn ich mit der Tool und der Trip kit config https://mpmediasoft.de/products/AIPBrowserDE/help/samples/trip-kit-config-AIP-VFR-Germany-ffcp.json versuche ein package zu erstellen, dann bricht das nach einer Zeit ab, weil es keinen Zugriff mehr auf eine bestimmte datei hat. Versuche ich diese manuel zu öffnen, erhalte ich von der dfs Seite einen Access Denied, auch ein Zugriff auf https://aip.dfs.de/basicAIP/ wird mit access denied beantwortet. ich vermute fast, dass dieser automatisierte Zugriff eine request limitation bewirkt. kann dies noch jemand beobachten?

**Comment by mipastgt** [comment]
Ja, die Beobachtung ist richtig. Siehe: https://github.com/mipastgt/AIPBrowserDE-Public/issues/19
Ich arbeite daran und kann wohl in den nächsten Tagen eine Lösung anbieten.

**Comment by mipastgt** [comment]
Eine Vorschau auf die kommende Version 1.2.0 des **AIP Browser DE**, die dieses Problem beheben sollte, kann über die Webseite
https://mpmediasoft.de/products/AIPBrowserDE/help/AIPBrowserDE-Preview.html 
heruntergeladen werden.

Ich würde mich freuen, wenn einige Leute diese Version ausprobieren und mir ein Feedback dazu geben würden. Wenn keine schwerwiegenden Probleme gefunden werden, wird diese Version die neue Release-Version werden.

**Reply by df1paw** [reply]
Sehr schön, danke!

Was ist denn mit dem Schloss-Symbol auf der Georefernz Seite passiert? Ich würde nämlich gerne noch zwei Seite der AIP selbst referenzieren (GEN 3-10 mit den Frequenzen für IFR/VFR Wechselverfahren und ENR 6 minimum vectoring altitude)

**Reply by mipastgt** [reply]
Diese Funktion ist leider dem Rotstift zum Opfer gefallen, da sie für den allgemeinen Gebrauch zu gefährlich und fehleranfällig ist. Man sieht das auch schon an diesem Beispiel. Die beiden genannten Karten lassen sich nämlich mit dem AIP Browser DE gar nicht georeferenzieren, da die Verzerrung der Karten sich nicht über eine rein affine Transformation auflösen läßt. (Nicht-parallele Längengrade) Und selbst wenn der AIP Browser DE das könnte, dann gäbe es immer noch keine Möglichkeit dies in eine der mir bekannten Flugplanungssoftwaren zu integrieren, da die das auch nicht können. Man müsste, damit das geht, die Karten in eine Merkator-Projection umtransformieren. Das ist zwar machbar, aber nur mit erheblichen Mehraufwand. Für den genannten Zweck würde ich andere Tools, wie z.B. GDAL oder QGIS vorschlagen. Die können das.
